bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -263,23 +271,37 @@ def __del__(self):
     def _gen_run_method(runnable_self: TensorflowRunnable, method_name: str):
         raw_method = getattr(runnable_self.model, method_name)
         method_partial_kwargs = partial_kwargs.get(method_name)
-        if method_partial_kwargs:
-            raw_method = functools.partial(raw_method, **method_partial_kwargs)
-
-        def _mapping(item: "TFArgType") -> "tf_ext.TensorLike":
-            if not LazyType["tf_ext.TensorLike"]("tensorflow.Tensor").isinstance(item):
-                return t.cast("tf_ext.TensorLike", tf.convert_to_tensor(item))
```

Comment:
I wonder if it is possible to also remove `typing.cast` when compiling to pyo @sauyon ?